protocols/imap/src/main/java/org/apache/james/imap/api/display/HumanReadableText.java

```diff
@@ -130,6 +130,7 @@ public static HumanReadableText permanentFlags(Flags flags) {
 
     public static final HumanReadableText BAD_IO_ENCODING = new HumanReadableText("org.apache.james.imap.BAD_IO_ENCODING", "failed. Illegal encoding.");
     public static final HumanReadableText COMPLETED = new HumanReadableText("org.apache.james.imap.COMPLETED", "completed.");
+    public static final HumanReadableText SUCCEEDED = new HumanReadableText("org.apache.james.imap.SUCCEEDED", "succeeded.");
```

Comment:
IMO both are the same. Why introduce a new one?

Reply:
From sample in https://datatracker.ietf.org/doc/html/rfc7628

Reply:
Yes but `t1 OK SASL authentication completed` is good too.

We don't need to have comment messages matching 100% ;-)